src/cases/containers/case-hearings/case-hearings.component.ts

```diff
@@ -53,6 +78,22 @@ export class CaseHearingsComponent implements OnInit, OnDestroy {
     }
   }
 
+  public convertServiceToVM(hearings: CaseHearingModel[]) {
```

Comment:
Could we change the method name from 'convertServiceToVM' to 'calculateMostRecentHearingDate', as this is not a service.